lib/tpcal.py

```diff
@@ -83,12 +86,11 @@ def __init__(self, points, calibrate, touch_count = 5):
         self.label_main = lv.label(lv.scr_act(), None)
 
         style_circ = lv.style_t()
-        lv.style_copy(style_circ, lv.style_pretty_color)
-        style_circ.body.radius = LV_RADIUS_CIRCLE
+        style_circ.init()
```

Comment:
You created a style but did not set any style attribute, so it has no effect.
Original style sets "radius"

Reply:
Ok, I added:
style_circ.set_radius(lv.STATE.DEFAULT,LV_RADIUS_CIRCLE)
... but I don't see any difference. Can we not just skip this?
Btw: lv.style_pretty_color does not exist in v7. Is there a replacement?

Reply:
There's no replacement; you would have to put a background yourself.

Reply:
> Can we not just skip this?

Maybe we can. Need to see how it looks.